projects/batfish/src/main/java/org/batfish/smt/LogicalGraph.java

```diff
@@ -59,7 +60,6 @@ boolean isEdgeUsed(Configuration conf, Protocol proto, LogicalEdge e) {
   /*
    * Find the router Id for the neighbor corresponding to a logical edge.
    */
-  @Nullable
   Long findRouterId(LogicalEdge e, Protocol proto) {
```

Comment:
`long`

Reply:
The method can return null in some cases, so I'm using a boxed value here.

Reply:
Sorry, I'm missing something. Why remove `@Nullable` then?

Reply:
Ah, whoops I got it backwards, thought it was adding it. Good catch.